content/Tjenester/fiksprotokoll/_index.md

```diff
@@ -166,10 +167,29 @@ Definisjon av `EksternProtokollKontoResponseEksternProtokollKontoResponse`:
 ```
 
 
-### Sende meldinger på konto 
-Benytt [Fiks IO](https://ks-no.github.io/tjenester/fiksprotokoll/fiksio) når meldinger skal sendes på konto.
-Dette er IDen til kontoen og er den samme som den tilhørende Fiks IO-kontoen. ID brukes når systemet skal sende meldinger fra kontoen, og når andre systemer skal sende meldinger til kontoen.
+### Sende og motta meldinger fra protokoll konto 
+Fiks protokoll bruker som tidligere nevnt Fiks IO til meldingsuveksling.
+Les mer om detaljene rundt meldingsutveksling på dokumentasjonssidene til [Fiks IO](https://ks-no.github.io/tjenester/fiksprotokoll/fiksio)
+
+### Overvåking
+Det anbefales at man overvåker at man har en fungerende mottakende komponent som henter meldinger fra køen. 
+Siden meldinger har en time-to-live kan man risikere at disse til slutt går ut på tid og man ikke får hentet dem.
+Det er selvfølgelig også i ens egen interesse å svare på meldinger så fort som mulig. 
+
+Hvordan man overvåker at man kan sende og motta meldinger er opp til en selv men vi anbefaler at man i det minste overvåker koblingsstatus til Fiks-IO for henting av meldinger. Klienten for .NET har f.eks. en IsOpen() metode som viser om det er en aktiv kobling.
+
+
+#### Status i fiks forvaltning
+Inne på forvaltningssidene til Fiks protokoll kan man se noen statuser på sine systemer og protokoll kontoer. En protokoll konto er som tidligere nevnt en wrapper rundt Fiks-IO, som igjen har en kø for meldinger til den kontoen.
+Enkel status for denne køen vises på forvaltningssidene.
+
+På system-siden vil man få en advarsel hvis man har protokollkonto(er) som ikke har kobling. Dette betyr at man har 1 eller flere protokollkonter som ikke lytter og henter meldinger fra køen.
+
+Hvis man går inn på listen over kontoer vil man også se status for hver konto, koblingsstatus som viser om det mangler noe som henter meldinger fra køen og antall meldinger som ligger på køen.
```

Comment:
_man også se status for hver konto,_ 

blir det riktigere å bytte ut komma med kolon?

man også se status for hver konto: